module/actor/actor-sheet.js

```diff
@@ -739,5 +806,100 @@ export async function setCovenant(selector, actor) {
         actorUpdate["data.covenant.value"] = found[0].value;
     }
 
+    await actor.update(actorUpdate);
+}
+
+export async function setWounds(selector, actor) {
+
+    const damageToApply = parseInt(selector.find('input[name$="damage"]').val())
+    const modifier = parseInt(selector.find('input[name$="modifier"]').val());
+
+    const damage = damageToApply - modifier - actor.data.data.combat.prot - actor.data.data.characteristics.sta.value;
+
+    const sizesAndWounds = 
```

Comment:
This should be put in a utility function that handle all cases (size lower than -4, bigger than 3, negative damage?)